csm/core/blogic/const.py

```diff
@@ -35,7 +35,7 @@
 DB_CONF_FILE_NAME = 'database.yaml'
 PLUGIN_DIR = 'cortx'
 WEB_DEFAULT_PORT = 28100 # currently being used by USL only
-PROVISIONER_LOG_FILE_PATH = "/var/log/seagate/provisioner"
+PROVISIONER_LOG_FILE_PATH = "/var/log/seagate"
```

Comment:
Can we read this path from the KV config passed during setup?

Reply:
No this is not Parsed in Config so can't fetch from that also this path is required by CSM so access provided is in post_install command itself.

Reply:
No this is a log path and not present in Conf Store also as per our dicussion the access for the path is required by CSM hence this should be handeled by CSM only.